packages/shared/lib/services/sync/sync.service.ts

```diff
@@ -639,11 +639,14 @@ export async function findDemoSyncs(): Promise<PausableSyncs[]> {
     return syncs;
 }
 
-export async function findRecentlyDeletedSync(): Promise<{ id: string; environmentId: number }[]> {
+export async function findRecentlyDeletedSync(): Promise<{ id: string; environmentId: number; connectionId: number; models: string[] }[]> {
     const q = db.knex
         .from('_nango_syncs')
-        .select<{ id: string; environmentId: number }[]>('_nango_syncs.id', '_nango_connections.environment_id')
+        .select<
+            { id: string; environmentId: number; connectionId: number; models: string[] }[]
+        >('_nango_syncs.id', '_nango_connections.environment_id', '_nango_connections.id', '_nango_sync_configs.models')
```

Comment:
_nango_connections.connection_id
```suggestion
        >('_nango_syncs.id', '_nango_connections.environment_id', '_nango_connections.connection_id', '_nango_sync_configs.models')
```

Reply:
it is actually `connection.id` (the internal number id, not the user-defined one).

Reply:
oh okay my bad, but they are conflicting no? both are returning `id` in the final json

Reply:
the column name of the result have indeed the same `id` name but knex map them to the select type `{ id: string; environmentId: number; connectionId: number; models: string[] }` based on column order so it works.
But you are right let's be explicit, I'll add aliases